doc/GAME_BALANCE.md

```diff
@@ -53,6 +54,20 @@ Notable skill: 4 (natural dodging ability; wolf, feral soldier; mi-go, vinebeast
 
 Maximal skill: 8 (highest for balance purposes; giant jumping spider, dermatik, liquid cat, atlantic salmon)
 
+
+# Monster armor scaling:
+
+Minimum armor: 0 (no armor at all; zombie, feral, blank body, giant fly, bulldog)
```

Comment:
I wouldn't include the giant fly here, because I disagree that it should have 0 armor. Any arthropod, even dog-sized, should have some armor on there, even if not much.